VideoWeb/VideoWeb/ClientApp/src/scripts/pexrtc.js

```diff
@@ -3568,9 +3575,9 @@ PexRTC.prototype.getPresentationURL = function() {
         if (self.png_presentation) {
             url = "https://" + self.node + "/api/client/v2/conferences/" + self.conference_uri + "/presentation.png?id=" + self.presentation_event_id + "&token=" + self.token;
         } else {
-	    if (self.bandwidth_in > 512) {
-		presentation_image = "presentation_high.jpeg";
-	    }
+            if (self.bandwidth_in > 384) {
```

Comment:
does this need to be configurable? where did 384 come from? :)

Reply:
This is the Pexip client code. I believe we do set the bandwidth in our component.
See `WaitingRoomBaseComponent` line 36 where to set the bandwidth to a constant value.